app/src/main/AndroidManifest.xml

```diff
@@ -3,19 +3,20 @@
     package="com.mutenlab.sudoit">
 
     <application
-        android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".SplashActivity">
```

Comment:
Remember that if the splash screen uses an especific background color or an immutable image, it should be set as theme and not in the activity layout.

Reply:
Sure thing. Applying changes